spec/support/question_set_helper.rb

```diff
@@ -1,5 +1,11 @@
 module QuestionSetHelper
-  VALID_ANSWERS = ['309 Colver Rd', '812', 'Jasper', '49230', 'None Of The Above']
+  VALID_ANSWERS = [
+    '309 Colver Rd',
+    '812',
+    'Jasper',
+    '49230',
+    'None Of The Above'
+  ]
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/59/files#r52251279